server/controllers/applicant/getApplicantsStats.js

```diff
@@ -0,0 +1,26 @@
+const Applicant = require('../../database/models/applicant');
+
+const getApplicantsStats = (req, res, next) => {
+  Promise.all([
+    Applicant.find({ accepted: true }).countDocuments(),
+    Applicant.find({
+      applicationSubmittedDate: { $exists: true, $ne: null },
```

Comment:
by using `{$exists:true}` This will return all documents with a key called "applicationSubmittedDate", but they may still have a null value, but `{$ne:null}` it will return all documents with both a key called "applicationSubmittedDate" and a non-null value , and that's exactly what we need to use 
```suggestion
     applicationSubmittedDate: { $ne: null },
```